app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/ControllerOptionsView.java

```diff
@@ -85,6 +90,19 @@ private void setScrollDirection(int checkedId, boolean doApply) {
         }
     }
 
+    private void setHapticFeedback(boolean value, boolean doApply) {
```

Comment:
I find this method's  functionality a bit cryptic. It's a setter, so I'd expect the boolean value to be the one to be set. However, it's just a flag to determine whether we set either the DEFAULT value or 0. Additionally, there is another flag in the 'doApply' parameter that it's difficult to know what it's used for unless you check the code. Apparently, it's just to indicate whether we are going to salve the value (not the one in the parameters (!!!)) into the settings store or not.

Reply:
Actually, this just follows the naming of `setCurvedDisplay` and many other naming. https://github.com/Igalia/wolvic/blob/f35e6b17ec7c0cd622b08a16ce5896df09bb65a4/app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/DisplayOptionsView.java#L218

If we are changing it to another one, I would recommend we change all of them together.

Reply:
umm, fair enough, but I think we should add a comment or file an issue to avoid forgetting about this. @svilllar, what's your opinion about this ?

Reply:
Yeah booleans are bad for APIs in general (things like `method(true,false)` are cryptic, error-prone and force you to check the method signature and even implementation). I agree with @javifernandez that we better use an enum type for them. However I also agree with @HollowMan6 that we can do it later for all of them at once.

Reply:
Okay, let's do this later